packages/plugin-npm/sources/index.ts

```diff
@@ -63,6 +63,7 @@ const plugin: Plugin = {
     npmRegistries: {
       description: `Settings per registry`,
       type: SettingsType.MAP,
+      normalizeKeys: key => key.replace(/\/$/, ``),
```

Comment:
I think it's better if you import `normalizeRegistry` from `npmConfigUtils` (it's fine to export it), this way there's no risk the two will ever diverge again.